MaxRectsBinPack.h

```diff
@@ -22,11 +22,11 @@ class MaxRectsBinPack
 	MaxRectsBinPack();
 
 	/// Instantiates a bin of the given size.
-	MaxRectsBinPack(int width, int height);
+	MaxRectsBinPack(int width, int height, bool allowFlip = true);
```

Comment:
Would be nice to have a docline "/// @param allowFlip Specifies whether the packing algorithm is allowed to rotate the input rectangles by 90 degrees to consider a better placement." or something like that?